JeffShepherd/BE-coding-challenge
# Feature/click counter

## What does this PR do?

- Adds main.js file and functions
- Adds tests for main.js functions

## How should it be tested?

- Review new tests and ensure the logic is valid
- Run 'npm test' and check that all tests pass
- Run 'node src/main.js' and ensure the console output is a sorted JSON array
